File: Bacharelado Ciência da Computação IMPACTA/3 Semestre/Análise e Projeto de Algoritmos/Lista/src/Lista/Exercício10.java
/*
Exercício 10 - Desenvolvido por Gabriel Muchon Pavanelli. Faculdade Impacta 19/02/2025.

10 - Faça um programa em Python que leia a largura e altura de uma parede, calcule e
mostre a área a ser pintada e a quantidade de tinta necessária para o serviço, sabendo
que cada litro de tinta pinta uma área de 2 metros quadrados.


 */
package Lista;

import java.util.Scanner;

public class Exercício10 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Seja bem-vindo ao programa de calcular a tinta necessária para seu serviço!");
        System.out.println("Digite a largura de sua parede: ");
        double largura = sc.nextDouble();
        System.out.println("Digite a altura de sua parede: ");
        double altura = sc.nextDouble();
        double area = largura * altura;
        double qtdTinta = area / 2;
        System.out.println("Você precisará de: " + qtdTinta + " Litros de tinta.");
        sc.close();
    }
}

File: Bacharelado Ciência da Computação IMPACTA/3 Semestre/Análise e Projeto de Algoritmos/Lista/Exercício10.py
"""
Exercício 10 - Desenvolvido por Gabriel Muchon Pavanelli. Faculdade Impacta 21/02/2025.

10 - Faça um programa em Python que leia a largura e altura de uma parede, calcule e
mostre a área a ser pintada e a quantidade de tinta necessária para o serviço, sabendo
que cada litro de tinta pinta uma área de 2 metros quadrados."""

largura = float(input("Digite a largura da sua parede: "))
altura = float(input("Digite a altura da sua parede:"))
area = largura * altura
qtdTinta = area / 2
print("Você precisará de: " , qtdTinta , " Litros de tinta.");
